README.md

```diff
@@ -16,7 +16,7 @@ This repository contains a simple sample project to help you getting started wit
 
 To complete this tutorial:
 
-* Install [Go](https://golang.org/dl/) 1.8 or later 
+* Install [Go](https://golang.org/dl/) 1.17.3 or later
```

Comment:
Nit: What feature requires 1.17?  Could we get away with saying 1.16 which is the current baseline for the SDK?  

What is the plan for these samples when we move to generics and require 1.18?

Reply:
Luckily, there are only a few existing service articles. When the change to 1.18 happens, let me know and I can get the versions updated. :) I'll start working on an include for that section so I can made one change that updates all docs.